server/index.js

```diff
@@ -1,19 +1,43 @@
 require('dotenv/config');
 const path = require('path');
 const express = require('express');
+const pg = require('pg');
 const errorMiddleware = require('./error-middleware');
+const ClientError = require('./client-error');
 
 const app = express();
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
 const publicPath = path.join(__dirname, 'public');
 
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
 if (process.env.NODE_ENV === 'development') {
   app.use(require('./dev-middleware')(publicPath));
 } else {
   app.use(express.static(publicPath));
 }
 
-app.get('/api/hello', (req, res) => {
-  res.json({ hello: 'world' });
+app.post('/api/workout', (req, res, next) => {
+  const { userId } = req.body;
```

Comment:
Instead of taking the user id as a body property, just hard code the user id in your SQL for now